src/components/Feedback.css

```diff
@@ -0,0 +1,60 @@
+.card1 {
+  display: flex;
+  flex-direction: column;
+  background-color: var(--dark-blue);
+  border-radius: 20px;
+  padding: 20px;
+  max-width: 350px;
+}
+
+.h3 {
+  color: var(--white);
+  margin-top: 20px;
+}
+
+p {
```

Comment:
This will apply styles to ALL `<p>` tags in your code, even if they're in different files,

For example `<App />` or `<Thanks />` . I suggest changing it to a class, or if it IS for all `<p>` tags, move it to `App.css`